tests/ccl_test_p2d.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np,math
+from numpy.testing import assert_,assert_raises, assert_almost_equal, assert_allclose, decorators, run_module_suite
+import pyccl as ccl
+from pyccl import CCLError
+
+def pk1d(k) :
+    return ((k+0.001)/0.1)**(-1)
+
+def grw(a) :
+    return a
+
+def pk2d(k,a) :
+    return pk1d(k)*grw(a)
+
+def lpk2d(k,a) :
+    return np.log(pk2d(k,a))
+
+def all_finite(vals):
+    """
+    Returns True if all elements are finite (i.e. not NaN or inf).
+    """
+    return np.all( np.isfinite(vals) )
+
+def check_p2d_init():
+    """
+    Check that background and growth functions can be run.
+    """
```

Comment:
Why is this function empty?

Reply:
Sorry, some residuals from copy-paste. Removed